api/controller.py

```diff
@@ -260,7 +257,7 @@ def load_settings(self):
         configuration after changes are made in the administrative
         interface.
         """
-        self.analytics = Analytics(self._db, refresh=True)
+        self.analytics = container_instance().analytics.analytics()
```

Comment:
This should get moved into the `__init__` function, since it does not need to be done every time the settings are reloaded anymore. The container also gets passed into `__init__`, so we should be using that instead of `container_instance()`.

Reply:
I have moved this to the `__init__` but noticed there is an instance of `Service` passed in as well, I don't think its used anywhere though, should I remove it?